x-pack/docs/en/rest-api/security/enroll-kibana.asciidoc

```diff
@@ -0,0 +1,51 @@
+[[security-api-kibana-enrollment]]
+=== Enroll Kibana API
+++++
+<titleabbrev>Enroll Kibana</titleabbrev>
+++++
+
+Allows a kibana instance to configure itself to communicate with a secured {es} cluster.
+
+[[security-api-kibana-enrollment-request]]
+==== {api-request-title}
+
+`GET /_security/enroll/kibana`
+
+[[security-api-kibana-enrollment-prereqs]]
+==== {api-prereq-title}
+
+
+[[security-api-kibana-enrollment-desc]]
+==== {api-description-title}
+
+The purpose of the enroll kibana API is to allow a kibana instance to configure itself to
+communicate with an {es} cluster that is already configured with security features
+enabled.
+
+[[security-api-client-enrollment-examples]]
+==== {api-examples-title}
+
+
+==== Example of enrolling a kibana instance
+
+[source,console]
+--------------------------------------------------
+GET /_security/enroll/kibana
+--------------------------------------------------
+// TEST[skip:we need to enable HTTP TLS for the docs cluster]
+
+[source,console_result]
+--------------------------------------------------
+{
+  "password" : "longsecurepassword", <1>
+  "http_ca" : "MIIJlAIBAzCCCVoGCSqGSIb3DQEHAaCCCUsEgglHMIIJQzCCA98GCSqGSIb3DQ....vsDfsA3UZBAjEPfhubpQysAICCAA=", <2>
+  "nodes_addresses" : [                                                                                          <3>
+    "192.168.1.2:9203"
+  ]
+}
+--------------------------------------------------
+<1> The password for the `kibana_system` user
+<2> The CA certificate that is used to sign the certificates that {es}
+nodes use for TLS on the HTTP layer as a Base64 encoded string of the ASN.1 DER encoding of the certificate
```

Comment:
```suggestion
<2> The CA certificate used to sign the node certificates that {es} uses for TLS on the HTTP layer.
The certificate is returned as a Base64 encoded string in ASN.1 DER encoding.
```

Reply:
I partially accepted this. The `ASN.1 DER` encoded version of the certificate _then_ gets Base64 encoded before we return it, your suggestion implies the opposite order